packages/plugins/minos-discovery-kong/minos/plugins/kong/client.py

```diff
@@ -130,8 +173,40 @@ async def activate_basic_auth_plugin_on_route(self, route_id: str):
             return resp
 
     async def activate_jwt_plugin_on_route(self, route_id: str):
-        payload = {"name": "jwt", "config": {"secret_is_base64": False, "run_on_preflight": True}}
+        payload = {
+            "name": "jwt",
+            "config": {"secret_is_base64": False, "run_on_preflight": True, "claims_to_verify": ["exp", "nbf"]},
+        }
 
         async with httpx.AsyncClient() as client:
             resp = await client.post(f"{self.route}/routes/{route_id}/plugins", json=payload)
             return resp
+
+    async def generate_jwt_token(
+        self, key: str, secret: str, algorithm: str = "HS256", exp: datetime = None, nbf: datetime = None
+    ) -> str:
+        payload = {"iss": key, "exp": exp, "nbf": nbf}
+
+        current = datetime.now(tz=utc)
```

Comment:
You can use `minos.common.current_datetime` instead of directly generating a timezone based datetime by your own.

Reply:
Done!